pyro/distributions/von_mises.py

```diff
@@ -41,6 +51,36 @@ def _log_modified_bessel_fn_0(x):
     return result
 
 
+def _fit_params_from_samples(samples, n_iter=50):
```

Comment:
If we are only using this for testing, let us move this to `test_von_mises.py`. If you have reason to think this will be more generally useful, we can add a docstring and remove the leading underscore.